<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distribución de Agua en Arequipa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Distribución de Agua Potable en Arequipa</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="optimizeButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Optimizar Distribución</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>0</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>721</width>
           <height>671</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/assets/mapa.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="Cercado">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>440</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="AltoSelvaAlegre">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>340</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Cayma">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>230</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="CerroColorado">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>290</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Sachaca">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>540</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Tiabaya">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>510</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Socabaya">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>550</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="JacoboHunter">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>530</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="JoseLuisBustamanteyRivero">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>490</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Yanahuara">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>460</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Paucarpata">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>460</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="MarianoMelgar">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>410</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Miraflores">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>370</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 150);  /* Fondo con color semi-transparente */
        border-radius: 15px;  /* Bordes redondeados */
        transition: all 0.3s ease;  /* Transición suave para todos los efectos */
}

QPushButton:hover {
        background-color: rgba(255, 255, 255, 200);  /* Fondo más opaco al pasar el ratón */
        border-radius: 15px;  /* Mantener bordes redondeados */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="verticalLine">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QGraphicsView" name="pieChartView">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="legendLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Leyenda</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="legendTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
